### PR TITLE
Added web-server with all calculator methods and updated setup.py

### PR DESCRIPTION
**Pull request by BasileoLuv**
ADSAFE-200 and ADSAFE-199(python part)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -12,12 +14,13 @@ def run(self, *args, **kwargs):
 
 setup(
     name="Python-Calculator",
-    version="0.2",
+    version=datetime.utcnow().isoformat(),
```
Comment (savermyas) [inline setup.py]: How will you make the release from such version?
It should be like "0.2-SNAPSHOT", where you substitute "SNAPSHOT" substring in job runtime, and during release just reove thist tail.

File: setup.py [hunk 2]
```diff
@@ -12,12 +14,13 @@ def run(self, *args, **kwargs):
...
     packages=['calc'],
     test_suite="tests",
     scripts=[
         'calculator_app.py',
         'service.py',
-        'README.md', ],
+        'README.md',
+        'pycodestyle.sh'],
```
Comment (savermyas) [inline setup.py]: You don't need to include the test in final package

File: setup.py (hunk 1 above)
Comment (BasileoLuv) [reply setup.py]: Fixed

**Review by savermyas — APPROVED** [review]
Resolve conflicts first.

**Pull request merged**